docs/content/reference/install-configuration/providers/docker.md

```diff
@@ -0,0 +1,441 @@
+
+!!! tip "The Quick Start Uses Docker"
+
+    If you have not already read it, maybe you would like to go through the [quick start guide](../../../getting-started/quick-start.md) that uses the Docker provider.
+
+## Configuration Example
+
+!!! info
+    This provider works with [Docker (standalone) Engine](https://docs.docker.com/engine/).
+
+Enabling the docker provider
+
+```yaml tab="File (YAML)"
+providers:
+  docker:
+    endpoint: "http://127.0.0.1:2375"
+```
+
+```toml tab="File (TOML)"
+[providers.docker]
+  endpoint = "http://127.0.0.1:2375"
+```
+
+```bash tab="CLI"
+--providers.docker.endpoint=tcp://127.0.0.1:2375
+```
+
+Attaching labels to containers (in your docker compose file)
+
+```yaml
+version: "3"
+services:
+  my-container:
+    # ...
+    labels:
+      - traefik.http.routers.my-container.rule=Host(`example.com`)
+```
+
+## Configuration Options
+
+| Field | Description                                               | Default              | Required |
+|:------|:----------------------------------------------------------|:---------------------|:---------|
+| `providers.providersThrottleDuration` | Minimum amount of time to wait for, after a configuration reload, before taking into account any new configuration refresh event.<br />If multiple events occur within this time, only the most recent one is taken into account, and all others are discarded.<br />**This option cannot be set per provider, but the throttling algorithm applies to each of them independently.** | 2s  | No |
+| `providers.docker.endpoint` | Specifies the Docker API endpoint. See [here](#endpoint) for more information|  `unix:///var/run/docker.sock`     | Yes   |
+| `providers.docker.username` | Defines the username for Basic HTTP authentication. This should be used when the Docker daemon socket is exposed through an HTTP proxy that requires Basic HTTP authentication.|  " "    | No   |
+| `providers.docker.password` | Defines the password for Basic HTTP authentication. This should be used when the Docker daemon socket is exposed through an HTTP proxy that requires Basic HTTP authentication.|  " "    | No   |
+| `providers.docker.useBindPortIP` | Instructs Traefik to use the IP/Port attached to the container's binding instead of its inner network IP/Port. See [here](#usebindportip) for more information |  false   | No   |
+| `providers.docker.exposedByDefault` | Expose containers by default through Traefik. See [here](./overview.md#restrict-the-scope-of-service-discovery) for additional information |  true    | No   |
+| `providers.docker.network` | Defines a default docker network to use for connections to all containers. This option can be overridden on a per-container basis with the `traefik.docker.network` label.|  ""    | No   |
+| `providers.docker.defaultRule` | Defines what routing rule to apply to a container if no rule is defined by a label. See [here](#defaultrule) for more information |  `Host(`{{ normalize .Name }}`)`   | No   |
```

Comment:
The rendering of the default value is broken here.

Reply:
Ah yes, i know what the issue is, will fix